sdk/src/utils/config.ts

```diff
@@ -1,224 +1,259 @@
 import { ethers } from "ethers";
-import { isAddress } from "ethers/lib/utils.js";
 
-import { DEFAULT_CHAIN_ID, DEPLOYMENTS } from "../constants.js";
+import { DEPLOYMENTS } from "../constants";
 import {
   ConfigurationError,
-  Deployment,
   HypercertClientConfig,
   InvalidOrMissingError,
+  Operator,
   SupportedChainIds,
   UnsupportedChainError,
-} from "../types/index.js";
-import logger from "./logger.js";
+} from "../types";
+import logger from "./logger";
 
 /**
  * Returns the configuration for the Hypercert client, based on the given overrides.
  * @param overrides An object containing overrides for the default configuration.
  * @returns The configuration for the Hypercert client.
  * @throws An `UnsupportedChainError` if the chain ID is not 5 or 10, or if the chain ID is missing or not found.
```

Comment:
These chain ids should probably be updated as we support more different chains now.

Reply:
Done, thanks